src/mgr/FunAction.cc

```diff
@@ -49,10 +48,12 @@ Mgr::FunAction::dump(StoreEntry* entry)
 {
     debugs(16, 5, MYNAME);
     Must(entry != nullptr);
-    if (UsingSmp())
-        storeAppendPrintf(entry, "by kid%d {\n", KidIdentifier);
+
+    // always non-aggregatable, no need to check
+    openKidSection(entry, is_yaml());
```

Comment:
I think with the long discussion you might have lost track of the context.

No reports using FunAction have the possibility of being aggregatable,
for the simple fact that they don't have the means to report that fact back
to the outside framework. Adding the aggregatable check here is
constexpr, but the compiler can't elide it because it's virtual.

However, FunAction is the most well developed of the Actions we have,
so others could do like I did, and use it as a blueprint, so leaving
some note for humans could help them avoid any pitfalls.

If we were int he context of Action, then the aggregatable() check would
be the right thing to do. But here it isn't.

In the interest of moving forward, I'll just remove the comment. Hopefully,
after I'll be done with updating the reports to yaml, no one else
will have to touch them for a long while.